profiles/thurloe-configs/terraform/thurloe/config.tf

```diff
@@ -0,0 +1,42 @@
+provider "vault" {}
```

Comment:
The update to `dsp-k8s-deploy` exposes `VAULT_TOKEN` and `VAULT_ADDR` in terraform; the vault provider [docs](https://www.terraform.io/docs/providers/vault/index.html#provider-arguments) recommend letting this provider use the env vars to configure itself rather than specify them.